resources/htmlTemplate/TocHTMLTemplate.html

```diff
@@ -134,20 +134,19 @@ <h1><a href="/" title="Home" rel="home">Software Package Data Exchange (SPDX)</a
 
       <h1>SPDX License List</h1>
 
-      <p>The SPDX License List is a list of commonly found licenses and exceptions used in free and open source and other collaborative software. The purpose of the SPDX License List is to enable 
-      easy and efficient identification of such licenses and exceptions in an SPDX document 
-      (or elsewhere). The SPDX License List includes a standardized short identifier, full name 
-      for each license, vetted license text, other basic information, and a canonical permanent 
-      URL for each license and exception. By providing a short identifier, users can efficiently 
-      refer to a license without having to redundantly reproduce the full license.  
+      <p>The SPDX License List is a list of commonly found licenses and exceptions used in free and open source and other collaborative software or documentation. The purpose of the SPDX License List is to enable 
+      easy and efficient identification of such licenses and exceptions in an SPDX document, in source files 
+      or elsewhere. The SPDX License List includes a standardized short identifier, full name, vetted license 
```

Comment:
I prefer “and” to your “or” here.  See also [this short discussion of similar usage][1].

[1]: https://github.com/spdx/license-list-XML/pull/538#discussion_r158381181